tgui/packages/tgui-panel/reconnect.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button } from 'tgui/components';
+
+let url: string | null = null;
+
+setInterval(() => {
+  Byond.winget('', 'url').then(currentUrl => {
+    // Sometimes, for whatever reason, BYOND will give an IP with a :0 port.
+    if (currentUrl && !currentUrl.match(/:0$/)) {
+      url = currentUrl;
+    }
+  });
+}, 5000);
+
+export const ReconnectButton = (props, context) => {
```

Comment:
This should probably be named ReconnectButtons

Reply:
done